File: Udemy/Algoritmos e Logica de Programacao/C++/Estrutura Sequencial/ProblemaIdades.cpp
/*Problema "idades"
Fazer um programa para ler o nome e idade de duas pessoas. Ao final mostrar uma mensagem com os
nomes e a idade média entre essas pessoas, com uma casa decimal, conforme exemplo.*/


#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
using namespace std;

int main(){
	
	string nome1, nome2;
	int idade1, idade2;
	double media;
	
	cout << "Dados da primeira pessoa: " << endl;
	cout << "Nome: ";
	getline(cin, nome1);	
	cout << "Idade: ";
	cin >> idade1;
	cout << "Dados da segunda pessoa: " << endl;
	cout << "Nome : ";
	cin.ignore(INT_MAX, '\n');
	getline(cin, nome2);

	cout << "Idade: ";
	cin >> idade2;
	media = (double)(idade1 + idade2)/2;
	cout << fixed << setprecision(1);
	cout << "A idade media de " << nome1 << " e " << nome2 << " e de " << media << " anos" << endl;

    return 0;
}

File: Udemy/Algoritmos e Logica de Programacao/Python/Estrutura Sequencial/ProblemaIdades.py
'''Problema "idades"
Fazer um programa para ler o nome e idade de duas pessoas. Ao final mostrar uma mensagem com os
nomes e a idade média entre essas pessoas, com uma casa decimal, conforme exemplo.'''

idade1: int; idade2: int; nome1: str; nome2: str;
media: float;

print("Dados da primeira pessoa: ")
nome1 = input("Nome: ")
idade1 = int(input("Idade: "))
print("Dados da segunda pessoa: ")
nome2 = input("Nome: ")
idade2 = int(input("Idade: "))

media = float(idade1 + idade2) / 2
print(f"A idade média de {nome1} e {nome2} é de {media:.1f} anos")